synapse/events/utils.py

```diff
@@ -494,7 +509,10 @@ def _inject_bundled_aggregations(
 
             # Don't bundle aggregations as this could recurse forever.
             serialized_latest_event = self.serialize_event(
-                thread.latest_event, time_now, bundle_aggregations=None
+                thread.latest_event,
+                time_now,
+                config=config,
```

Comment:
I don't know if this is correct or if we should only pass in the format, similar to what we do for redactions: https://github.com/matrix-org/synapse/blob/45f45404de2d0c4d68954eddc2dc905e50dfafe9/synapse/events/utils.py#L349-L352

Reply:
This was discussed a bit in #synapse-dev:matrix.org and we agree that things should definitely be consistent. @erikjohnston thinks that:

> Yeah, my natural instinct is that we should propagate the options for both. In practice I doubt it matters tooo much

I'm going to make that change here.